Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 20
Rules claiming more threads will be scaled down.
Conda environments: ignored
Singularity containers: ignored
Job counts:
	count	jobs
	1	all
	3	bgzip
	3	snpEff
	1	snpEffDbDownload
	8

[Sat Sep 12 22:23:50 2020]
rule snpEffDbDownload:
    output: resources/snpeff/
    log: logs/snpEff/snpeff_download.log
    jobid: 61

java -jar workflow/scripts/snpEff/snpEff.jar download -dataDir resources/snpeff/ Aedes_aegypti_lvpagwg 2> logs/snpEff/snpeff_download.log
Warning: the following output files of rule snpEffDbDownload were not present when the DAG was created:
{'resources/snpeff/'}
[Sat Sep 12 22:24:05 2020]
Finished job 61.
1 of 8 steps (12%) done

[Sat Sep 12 22:24:05 2020]
rule snpEff:
    input: results/variants/variants.3.vcf, resources/snpeff/
    output: results/variants/annot.variants.3.vcf, results/variants/snpeff/3.html, results/variants/snpeff/3.csv
    log: logs/snpEff/snpeff_3.log
    jobid: 47
    wildcards: chrom=3

[Sat Sep 12 22:24:05 2020]
rule snpEff:
    input: results/variants/variants.1.vcf, resources/snpeff/
    output: results/variants/annot.variants.1.vcf, results/variants/snpeff/1.html, results/variants/snpeff/1.csv
    log: logs/snpEff/snpeff_1.log
    jobid: 43
    wildcards: chrom=1

[Sat Sep 12 22:24:05 2020]
rule snpEff:
    input: results/variants/variants.2.vcf, resources/snpeff/
    output: results/variants/annot.variants.2.vcf, results/variants/snpeff/2.html, results/variants/snpeff/2.csv
    log: logs/snpEff/snpeff_2.log
    jobid: 45
    wildcards: chrom=2

/home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmpjpvcy2k0.wrapper.py
/home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmpj9umyavt.wrapper.py
/home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmpc6gyt8ya.wrapper.py
[Sat Sep 12 22:24:08 2020]
Error in rule snpEff:
    jobid: 45
    output: results/variants/annot.variants.2.vcf, results/variants/snpeff/2.html, results/variants/snpeff/2.csv
    log: logs/snpEff/snpeff_2.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 164 of /home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk:
Command 'set -euo pipefail;  /home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmpj9umyavt.wrapper.py' returned non-zero exit status 1.
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2189, in run_wrapper
  File "/home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk", line 164, in __rule_snpEff
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 529, in _callback
  File "/home/snagi/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 515, in cached_or_run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2201, in run_wrapper
Removing output files of failed job snpEff since they might be corrupted:
results/variants/annot.variants.2.vcf
[Sat Sep 12 22:24:08 2020]
Error in rule snpEff:
    jobid: 43
    output: results/variants/annot.variants.1.vcf, results/variants/snpeff/1.html, results/variants/snpeff/1.csv
    log: logs/snpEff/snpeff_1.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 164 of /home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk:
Command 'set -euo pipefail;  /home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmpc6gyt8ya.wrapper.py' returned non-zero exit status 1.
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2189, in run_wrapper
  File "/home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk", line 164, in __rule_snpEff
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 529, in _callback
  File "/home/snagi/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 515, in cached_or_run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2201, in run_wrapper
Removing output files of failed job snpEff since they might be corrupted:
results/variants/annot.variants.1.vcf
[Sat Sep 12 22:24:08 2020]
Error in rule snpEff:
    jobid: 47
    output: results/variants/annot.variants.3.vcf, results/variants/snpeff/3.html, results/variants/snpeff/3.csv
    log: logs/snpEff/snpeff_3.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 164 of /home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk:
Command 'set -euo pipefail;  /home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmpjpvcy2k0.wrapper.py' returned non-zero exit status 1.
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2189, in run_wrapper
  File "/home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk", line 164, in __rule_snpEff
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 529, in _callback
  File "/home/snagi/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 515, in cached_or_run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2201, in run_wrapper
Removing output files of failed job snpEff since they might be corrupted:
results/variants/annot.variants.3.vcf
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/snagi/lstm_data/rna-seq-ir/.snakemake/log/2020-09-12T222349.665650.snakemake.log
